checker/checker.go

```diff
@@ -23,9 +23,28 @@ type Report struct {
 	Errors []Error
 }
 
+// Example shows how to adhere and not adere to a rule.
+type Example struct {
+
+	// Good is an example of sticking to the rule.
+	Good string
+
+	// Bad is a counter-example that shows how to not apply the rule.
```

Comment:
How about `Bad is a counter-example showing a mis-use which lingo will report. ` or something along these lines?

Reply:
Done.